src/components/Button.tsx

```diff
@@ -176,13 +176,13 @@ export default function ({
     >
       {loading && <Loading small={small} />}
       {showContent && (
-        <div className={space('space-x-2')}>
+        <>
```

Comment:
We already have `space-x-2` inside the `<button>` styles, but it didn't work because...